tests/withKlaviyoAndroid.test.ts

```diff
@@ -0,0 +1,304 @@
+import * as fs from 'fs';
```

Comment:
sorry to be picky, but this seems like a fair few android-related changes for a PR titled "ios plugin unit testing"